src/designer/elsa-workflows-studio/src/components/designers/tree/elsa-designer-tree/models.ts

```diff
@@ -22,3 +22,14 @@ export enum LayoutDirection {
   RightLeft = 'rightleft',
   BottomTop = 'bottomtop'
 }
+
+export interface LayoutState {
+  dragging: boolean,
+  nodeDragging: boolean,
+  ratio: number,
+  scale: number,
+  left: number,
+  top: number,
+  initX: number,
+  initY: number
+}
```

Comment:
Should this interface be part of the new `elsa-designer` component, rather than being part of the current `elsa-designer-tree` component (which is not supposed to be changed in the first place)?